pkg/aws/client.go

```diff
@@ -140,6 +140,7 @@ func (b *ClientBuilder) Build() (Client, error) {
 			MaxRetries: aws.Int(25),
 			// Set MinThrottleDelay to 1 second
 			Retryer: client.DefaultRetryer{
+				NumMaxRetries:    5,
```

Comment:
I've tested it with 2 different accounts. One is my personal and the other what @fahlmant gave me. The retry is not a permanent solution I agree, but still, the number of 400s significantly reduced(I didn't see in the logs any but still can occur..). Having said that, there's no response code that `resp` contains (`resp, err := client.iamClient.SimulatePrincipalPolicy(input)`) It only fills the err message with status code when anything happens except 200. Because of that, it's hard to adjust the retry numbers dynamically if continuously 400 happens

Reply:
what could be the other alternatives? We're making a single request for policy checks so I don't think we can reduce the number of requests at all.

Reply:
Ok no problem, lets see how thing goes and look at it closer if we continue to see rate limiting errors.